cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/config/KafkaCruiseControlConfig.java

```diff
@@ -488,6 +488,14 @@
   private static final String GOALS_DOC = "A list of case insensitive goals in the order of priority. The high "
       + "priority goals will be executed first.";
 
+  /**
+   * <code>intra.broker.goals</code>
+   */
+  public static final String INTRA_BROKER_GOALS_CONFIG = "intra.broker.goals";
+  private static final String INTRA_BROKER_GOALS_DOC = "A list of case insensitive intra-broker goals in the order of priority. "
+      + "The high priority goals will be executed first. The intra-broker goals are only relevant if intra-broker operation is "
+      + "supported(i.e. in migrate_to_kafka_2_0 branch), otherwise this list should be empty. ";
```

Comment:
I think it would be better to indicate CC version rather than Github branch name (because we may change the branch name in the future) here -- i.e. maybe we can say this is supported only in CC versions above `2.*`?